massa-protocol-worker/src/protocol_controller.rs

```diff
@@ -92,189 +92,17 @@ pub async fn start_protocol_controller(
     ))
 }
 
-//put in a module to block private access from Protocol_worker.
-mod nodeinfo {
```

Comment:
Working on this PR, I spotted that this module is just dead code duplicated in `massa-protocol-exports` ... :smiling_face_with_tear:

Reply:
Feel free to remove dead code :)
Why doesn't the compiler warn about it tho ?

Reply:
:fearful:

Reply:
My bad, it was the whole file that was duplicated and not included in the project ...